styles.css

```diff
@@ -30,9 +30,58 @@ body {
   color: #83fe00;
   text-decoration: none;
 }
+
+#static-caption {
```

Comment:
You need to commit to a casing convention. You should use all `kebab-case` for your ids. some of them are `camelCase`.